documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -21,7 +21,27 @@ endif::[]
 [id="about-mtv"]
 == About {the-lc} {project-full}
 
-You can migrate virtual machines from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+You can migrate virtual machines from VMware vSphere, {rhv-full}, or {osh\p} source providers to {virt} with {the-lc} {project-first}.
+
+Migration using OpenStack source providers is a Technology Preview.
+
+[IMPORTANT]
+====
+Migration using OpenStack source providers is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs) and
+might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+//[NOTE]
+// ====
+//Migration using OpenStack source providers only supports VMs whose volume snapshots are supported by OpenStack Block Storage (Cinder). Migrations for VMs that use OpenStack Swift at all or use only OpenStack Glance for backend storage are not currently supported.
```

Comment:
I think we need to rephrase here, the requirement is that the cinder backend for the VMs volumes supports snapshot creation, and has it enabled

Reply:
@RichardHoch

Reply:
@RichardHoch